modules/getting-started/proc-customize-rhdh-homepage.adoc

```diff
@@ -22,10 +35,13 @@ proxy:
 	<HEADER_KEY>: <HEADER_VALUE> # optional and can be passed as needed i.e Authorization can be passed for private GitHub repo and PRIVATE-TOKEN can be passed for private GitLab repo
 ----
 
-* A separate service that provides the Home page data in JSON format using an API. When using a separate service, you can do the following:
-** Use the same service to provide the data to all configurable {product-short} pages or use a different service for each page.
-** Use the https://github.com/redhat-developer/red-hat-developer-hub-customization-provider[`red-hat-developer-hub-customization-provider`] as an example service, which provides data for both the Home and Tech Radar pages. The `red-hat-developer-hub-customization-provider` service provides the same data as default {product-short} data. You can fork the `red-hat-developer-hub-customization-provider` service repository from GitHub and modify it with your own data, if required.
-** Deploy the `red-hat-developer-hub-customization-provider` service and the {product-short} Helm chart on the same cluster.
+== Using a dedicated service to provide data to the Learning Paths
+
+When using a dedicated service, you can do the following:
+
+* Use the same service to provide the data to all configurable {product-short} pages or use a different service for each page.
+* Use the https://github.com/redhat-developer/red-hat-developer-hub-customization-provider[`red-hat-developer-hub-customization-provider`] as an example service, which provides data for both the Home and Tech Radar pages. The `red-hat-developer-hub-customization-provider` service provides the same data as default {product-short} data. You can fork the `red-hat-developer-hub-customization-provider` service repository from GitHub and modify it with your own data, if required.
+* Deploy the `red-hat-developer-hub-customization-provider` service and the {product-short} Helm chart on the same cluster.
```

Comment:
[SME] can you confirm that these options apply to Home page, Tech Radar, and Learning Paths config, and not just to Home page config only. 

I reformatted this existing content as part of this PR, but it only exists in the Home page config doc file, currently.

Reply:
yeah it applies to all